Updating for allow URL and file allow like this (#7869)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/manage-tenant-allows.md

> [!div class="mx-imgBorder"]
> ![False positive submission example.](../../media/admin-submission-allow-messages.png)

## Add URL allows using the Submissions portal 

Allow URLs in the Submissions section of Microsoft 365 Defender. 

1. In the Microsoft 365 Defender portal, go to **Actions & submissions** \> **Submissions**.

2. On the **Submissions** page, verify that the **URLs** tab is selected, and then click ![Ad icon.](../../media/m365-cc-sc-create-icon.png) **Submit to Microsoft for analysis**.

> 3. Use the **Submit to Microsoft for review** flyout to submit a message, either by adding the URL. 

Reviewer: ```suggestion
3. Use the **Submit to Microsoft for review** flyout to submit a message by adding the URL. 
```

4. In the **Select a reason for submitting to Microsoft** section, select **Should not have been blocked (false positive)**. 

> 5. Turn on **Allow URLs like this** option. 

Reviewer: ```suggestion
5. Turn on the **Allow URLs like this** option. 
```

> 6. From the **Remove after** drop-down list, specify how long you want the allow option to work.

Reviewer: ```suggestion
6. From the **Remove after** drop-down list, specify for how long you want the allow option to work.
```

7. When you're finished, click the **Submit** button.

## Add File allows using the Submissions portal 

Allow Files in the Submissions section of Microsoft 365 Defender. 

1. In the Microsoft 365 Defender portal, go to **Actions & submissions** \> **Submissions**.

2. On the **Submissions** page, verify that the **Email attachments** tab is selected, and then click ![Ad icon.](../../media/m365-cc-sc-create-icon.png) **Submit to Microsoft for analysis**.

> 3. Use the **Submit to Microsoft for review** flyout to submit a message, either by adding the file(s). 

Reviewer: ```suggestion
3. Use the **Submit to Microsoft for review** flyout to submit a message by adding the file or files. 
```

4. In the **Select a reason for submitting to Microsoft** section, select **Should not have been blocked (false positive)**. 

> 5. Turn on **Allow files like this** option. 

Reviewer: ```suggestion
5. Turn on the **Allow files like this** option. 
```

> 6. From the **Remove after** drop-down list, specify how long you want the allow option to work.

Reviewer: ```suggestion
6. From the **Remove after** drop-down list, specify for how long you want the allow option to work.
```
